dart/lib/flat_buffers.dart

```diff
@@ -1241,3 +1246,72 @@ class _VTable {
     }
   }
 }
+
+// Allocator interface. This is flatbuffers-specific - only for [Builder].
```

Comment:
Any public class should have a top level document comment `///` that is made up of complete sentences explaining its usage.

For example,

```
/// The interface that [Builder] uses to allocate buffers for encoding.
```

Reply:
Suggestion applied